src/mixins/itext_behavior.mixin.js

```diff
@@ -341,10 +341,28 @@
       this._setEditingProps();
 
       this._tick();
-      this.canvas && this.canvas.renderAll();
-
       this.fire('editing:entered');
-      this.canvas && this.canvas.fire('text:editing:entered', { target: this });
+      if (this.canvas) {
```

Comment:
you mean the "function(options) {}" ?

i just moved it from other section. i was enough happy that it was already working.

anyway an early return is something to do if !this.canvas